chemprop/uncertainty/predictor.py

```diff
@@ -46,11 +48,24 @@ def _calc_prediction_uncertainty(
 
 @UncertaintyPredictorRegistry.register("ensemble")
 class EnsemblePredictor(UncertaintyPredictor):
-    def _calc_prediction_uncertainty(
-        self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer
-    ) -> tuple[Tensor, Tensor]:
-        ...
-        return
+    """
+    Class that predicts uncertainty for predictions based on the variance in predictions among
+    an ensemble's submodels.
+    """
+    def _calc_prediction_uncertainty(self, dataloader, models, trainer) -> Tensor:
+        if len(models) <= 1:
+            raise ValueError(
+                "Ensemble method for uncertainty is only available when multiple models are provided."
+            )
+        ensemble_preds = []
+        for model in models:
+            preds = torch.concat(trainer.predict(model, dataloader), 0)
+            if isinstance(model.predictor, MulticlassClassificationFFN):
+                preds = torch.argmax(preds, dim=-1)
```

Comment:
```suggestion
```
This is not needed anymore after #987. Also, based on the discussion in the dev meeting, we want to return the prediction in the shape o `m x n x t x c` (#986).